diff --git a/mld/train_mvae.py b/mld/train_mvae.py
index 2457392..2704791 100644
--- a/mld/train_mvae.py
+++ b/mld/train_mvae.py
@@ -27,6 +27,8 @@ from diffusion.nn import mean_flat, sum_flat
 
 debug = 0
 
+os.environ['CUDA_VISIBLE_DEVICES'] = '3'
+
 @dataclass
 class VAEArgs:
     arch: str = "all_encoder"
@@ -128,7 +130,7 @@ class Args:
 
     track: int = 1
     wandb_project_name: str = "mld_vae"
-    wandb_entity: str = "interaction"
+    wandb_entity: str = "MangoBox"
 
 class Trainer:
     def __init__(self, args: Args):
